charts/k8s-monitoring/default_allow_lists/alloy_integration.yaml

```diff
@@ -34,8 +35,13 @@
 - prometheus_target_sync_length_seconds_sum
 - prometheus_wal_watcher_current_segment
 - traces_exporter_send_failed_spans
+- traces_exporter_send_failed_spans_total
 - traces_exporter_sent_spans
+- traces_exporter_sent_spans_total
 - traces_loadbalancer_backend_outcome
 - traces_loadbalancer_num_backends
 - traces_receiver_accepted_spans
+- traces_receiver_accepted_spans_total
 - traces_receiver_refused_spans
+- traces_receiver_refused_spans_total
+- up
```

Comment:
`up` is automatically added to all allow lists in the config
```suggestion
```

Reply:
https://github.com/grafana/k8s-monitoring-helm/blob/main/charts/k8s-monitoring/templates/alloy_config/_alloy.alloy.txt#L72